lib/challenge.js

```diff
@@ -18,9 +18,18 @@
   // - brake
   // - park
 
-// Answer Starts Here
-
-// Answer Ends Here
+// 'Calendar'
+// Description: Calendar keeps track of events and appointments.
+// Every 'calendar' has
+// -month
+// -year
+// -date
+// -time
+// Every 'calendar' can
+// set appointment/event reminders
+// set number of days per week (ie weekdays vs. full week)
+// set particular day of week as start of week
+// invite other users to attend appointments
```

Comment:
This is a reasonable abstraction for this particular problem, but we were looking for _three_. Can you think of two others?